bulk-update/bulk-update.js

```diff
@@ -97,7 +113,16 @@ export async function loadListData(source, fetchFunction = fetch, fetchWaitMs =
  */
 export default async function main(config, migrate, reporter = null) {
   config.reporter = reporter || config.reporter;
-  const { length } = config.list;
+  const { list, outputDir, siteUrl, stagePath, locales } = config;
+  const { length } = list;
+
+  if (outputDir) {
+    fs.mkdirSync(outputDir, { recursive: true });
+    fs.writeFileSync(`${outputDir}/list.json`, JSON.stringify(config.list, null, 2));
+
+    const output = list.map((entry) => [`${siteUrl}${entry}`, localizedStageUrl(siteUrl, entry, stagePath, locales)]);
```

Comment:
Better variable name might be `stageOutput` or `stageList`

Reply:
Sounds good!